src/libs/telemetry.js

```diff
@@ -29,7 +29,7 @@ export default class Telemetry {
     }
   }
 
-  static async send(event, extra_data) {
+  static send(event, extra_data) {
```

Comment:
I wonder if `async` may be useful for readability in such cases.  
Here, it makes explicit that `send()` returns a Promise (returned by `Ajax.post`)

Reply:
Actually I wonder too…
I don't find this system very reliable as nothing ensures this is systematic. You would have to know the full call chain of every function to properly tag it `async` or not.

But there is also the fact that `async` ensures that the return value **is** a Promise, warping any non-promise value in a new promise if needed. So, technically, this syntax-only PR may break something if some parts of the code rely on being always able to do a `.then()` on the return value of this function. I'm pretty sure there is no such code, but I wonder if this was used intentionnaly for now.